src/main/java/Duke.java

```diff
@@ -128,13 +143,14 @@ public static void main(String[] args) {
                 printHorizontalLine();
             }
             try {
-                if(listEditableCommands.contains(type)){
-                    WriteFile.writeToFile("data\\duke.txt",taskArrayList);
-                }
-                else{
+                if (listEditableCommands.contains(type)) {
+                    WriteFile.writeToFile("data\\duke.txt", taskArrayList);
+                } else {
                     continue;
                 }
-            } catch(IOException e){};
+            } catch (IOException e) {
+            }
+            ;
```

Comment:
The catch block is empty here, you should consider printing/ returning the error caught